recipes/dsnparse/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "dsnparse" %}
+{% set version = "0.1.11" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 6252d2a0b12e7c5bf1775e77ed0d3ee0a06f92937cdb2d2cac6ee039b557d0b4
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - python
+
+test:
+  imports:
+    - dsnparse
+
+about:
+  home: http://github.com/Jaymon/dsnparse
+  license: MIT
+  license_family: MIT
+  license_file: 
```

Comment:
Please, add the content of the ``license_file``

Reply:
Thanks, I had missed that! Fixed